tests/cephadm/test_osd_memory_target_cephadm.py

```diff
@@ -38,7 +38,7 @@ def run(ceph_cluster, **kw):
     log.info(f"Memory target is set on host '{hostname}' : '{value}'")
```

Comment:
I see these tests are used in RADOS test suites [1] as well, if renamed it will broke these test suites, Update test suites accordingly.
[1] https://github.com/red-hat-storage/cephci/blob/master/suites/pacific/rados/tier-2_rados_test-osd-rebalance.yaml#L94

Reply:
We can let the RADOS test suite files as is because they are under different directory and once the rename for the cephadm test is done then the naming will be different for both and they will execute the appropriate tests.

Reply:
ACK